src/battle.py

```diff
@@ -229,6 +313,17 @@ def get_melee_weapon(self):
         else:
             return self.weapons[0]
 
+    def switch_weapons(self):
+        temp = self.weapons[0] 
+        self.weapons[0] = self.weapons[1]
+        self.weapons[1] = temp
```

Comment:
Not a big deal, but could do a swap more easily with:

`self.weapons[0], self.weapons[1] = self.weapons[1], self.weapons[0]`